spec.html

```diff
@@ -784,23 +817,35 @@ <h1>ToElementDescriptor ( _elementObject_ )</h1>
         1. If _kind_ is not one of `"method"` or `"field"`, throw a *TypeError* exception.
         1. Let _key_ be ? Get(_elementObject_, `"key"`).
         1. If _key_ has a [[PrivateName]] internal slot, set _key_ to _key_.[[PrivateName]].
+        1. If _kind_ is `"initializer"`,
+          1. If _key_ is not *undefined*, throw a *TypeError* exception.
```

Comment:
Maybe we should throw _before_ converting `key` to a valid property key?

Reply:
Yes of course; also this broke the if/else conditional!